src/game.cpp

```diff
@@ -1823,7 +1823,7 @@ void Game::playerMove(uint32_t playerId, Direction direction)
 	player->resetIdleTime();
 	player->setNextWalkActionTask(nullptr);
 
-	player->startAutoWalk(std::forward_list<Direction> { direction });
+	player->startAutoWalk(std::vector<Direction> { direction });
```

Comment:
```suggestion
	player->startAutoWalk({ direction });
```

Should work as well

Reply:
Fixed. Added a version which takes Direction as parameter instead of requiring to create a new vector just for one direction(copied from @SaiyansKing).